### PR TITLE
map login packets

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

File: mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping [hunk 1]
```diff
@@ -2,9 +2,13 @@ CLASS dcp net/minecraft/client/network/ClientLoginNetworkHandler
...
 	FIELD c parentGui Lcut;
+	FIELD d statusConsumer Ljava/util/function/Consumer;
 	FIELD e connection Liu;
 	FIELD f playerProfile Lcom/mojang/authlib/GameProfile;
 	METHOD <init> (Liu;Lcqn;Lcut;Ljava/util/function/Consumer;)V
 		ARG 1 connection
 		ARG 2 client
 		ARG 3 parentGui
+	METHOD a loginServer (Ljava/lang/String;)Ljg;
+		ARG 1 serverid
```
Comment (Gegy) [inline mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: `serverId`?

Comment (Runemoro) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: Also, shouldn't the method be `loginToServer`/`login`/`onServerLogin`? The current name is implying that the server is the one being logged in.

Comment (asiekierka) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: @gegy1000 This is based on AuthLib's Javadoc.

@Runemoro You're right.

Comment (Gegy) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: It's still incorrect to Java standard though, is it not?

Comment (asiekierka) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: ```     * @param serverId The random ID of the server to join```

Comment (Gegy) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: Oh, I'm referencing the capitalisation of the `i` in `id`

Comment (asiekierka) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: Again, I'm sticking to the Javadocs here, which i think is *slightly* less confusing.

(Also,we tend to capitalize the **i** in our mappings)

Comment (Gegy) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: The mapping is `serverid`, it should be `serverId`

Comment (asiekierka) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: Oh! I'm really not well today.

Comment (Gegy) [reply mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping]: Hehe, should've clarified better.

**Review by Gegy — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Pull request merged**